gen-client.sh

```diff
@@ -100,7 +103,7 @@ FIX_TASK_ERROR_FILTER='(.components.schemas.TaskResponse|select(.)|.properties.e
 
 if [ "$LANGUAGE" = "python" ]
 then
-  cat "${API_SPEC}" | jq "." > patched-api.json
+  cat "${API_SPEC}" | jq "${FIX_TASK_CREATED_RESOURCES_FILTER}|${FIX_TASK_ERROR_FILTER}" > patched-api.json
```

Comment:
We should probably fix this in pulpcore instead. Now that we don't need to keep compatibility with old branches.

Reply:
https://github.com/pulp/pulpcore/pull/6111